library/src/skins/default/src/sass/base/_defaults.scss

```diff
@@ -176,13 +176,21 @@ input[type="checkbox"]{
     background-repeat: no-repeat;
     background-position: 1px 1px;
   }
+  &:disabled{
+    cursor: not-allowed;
+    background-color: var(--sakai-background-color-3);
+  }
```

Comment:
have you tested in dark mode?

Reply:
Yes. I've attached a screenshot below:

![DarkMode-Calendar-DisabledCheckboxes](https://github.com/user-attachments/assets/898b236f-0e03-4a19-b94e-a150ccf5ba4b)